testutils/src/main/java/com/amplifyframework/testutils/sync/SynchronousAuth.java

```diff
@@ -252,11 +254,52 @@ public Object confirmResetPassword(
      */
     @NonNull
     public AuthSession fetchAuthSession() throws AuthException {
-        return Await.<AuthSession, AuthException>result(AUTH_OPERATION_TIMEOUT_MS, (onResult, onError) ->
-                asyncDelegate.fetchAuthSession(onResult, onError)
+        return Await.result(AUTH_OPERATION_TIMEOUT_MS, asyncDelegate::fetchAuthSession);
+    }
+
+    /**
+     * Remembers current device synchronously.
+     * @return Dummy object - just indicates it completed successfully
+     * @throws AuthException exception
+     */
+    public Object rememberDevice() throws AuthException {
```

Comment:
Since we have a `forceDevice(AuthDevice specificDevice)`, I suggest that we call this `rememberCurrentDevice()`. (Same for Rx bindings, of course, as well.)